### PR TITLE
Lots of copy updates

### PR DESCRIPTION
**Pull request by schlosser**
Still a few more things needed but majority of changes from webapp text doc.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -40,8 +40,8 @@
     <!-- BUG: Note that these don't currently change per-page. -->
     <!-- https://github.com/covid-projections/covid-projections/issues/450 -->
     <meta data-react-helmet="true" property="og:url" content="https://covidactnow.org" />
-    <meta data-react-helmet="true" property="og:title" content="Real-time models and metrics to understand where we stand against COVID." />
-    <meta data-react-helmet="true" property="og:description" content="50 states. 2,100+ counties. Click the map to dive in." />
+    <meta data-react-helmet="true" property="og:title" content="Data to reopen safely." />
+    <meta data-react-helmet="true" property="og:description" content="@CovidActNow has crucial, real-time modeling and metrics to help America reopen safely. See how your community is going at covidactnow.org." />
```
Comment (mikelehen) [inline public/index.html:44]: "doing" => "going" ?

File: src/components/Header/HomePageHeader.tsx [hunk 2]
```diff
@@ -36,8 +36,9 @@ const HomePageHeader = () => {
         </HeaderTitle>
         <div>
           <HeaderSubCopy color="inherit" component="p" variant="subtitle2">
-            Real-time models and metrics to see where we stand against COVID and
-            help us reopen. All 50 states. 2,100+ counties.
+            Real-time models and metrics to see where we stand against COVID.{' '}
```
Comment (mikelehen) [inline src/components/Header/HomePageHeader.tsx:39]: "models" => "modeling" to match elsewhere?

File: src/enums/zones.ts [hunk 3]
```diff
@@ -2,14 +2,20 @@ import { COLOR_MAP } from './interventions';
...
+  CASE_GROWTH_RATE = 'Infection growth rate',
+  POSITIVE_TESTS = 'Positive test rate',
   HOSPITAL_USAGE = 'COVID ICU usage',
 }
 
+export const ChartTypeToMetricName = {
+  [ChartType.CASE_GROWTH_RATE]: 'Infection growth rate',
+  [ChartType.POSITIVE_TESTS]: 'Positive test rate',
+  [ChartType.HOSPITAL_USAGE]: 'COVID ICU usage',
+};
```
Comment (mikelehen) [inline src/enums/zones.ts:14]: Maybe I'm blind, but how is this different than the ChartTypeToTitle mapping?  Are we expecting them to diverge?  Or if this is a "we should clean this up" thing, can you add a TODO?

File: src/components/LocationPage/ChartsHolder.tsx [hunk 4]
```diff
@@ -127,11 +122,7 @@ const ChartsHolder = (props: {
                     }
                   />
                 </ZoneChartWrapper>
-                <Disclaimer>
-                  We define <strong>COVID ICU usage</strong> as the percentage
-                  of typically available ICU beds currently occupied by COVID
-                  patients.
-                </Disclaimer>
+                <Disclaimer metricName="COVID ICU usage" />
```
Comment (goldblatt) [inline src/components/LocationPage/ChartsHolder.tsx:125]: fwiw i really liked these definitions

File: src/enums/zones.ts [hunk 5]
```diff
@@ -2,14 +2,20 @@ import { COLOR_MAP } from './interventions';
 import { fail } from 'assert';
 
 export enum ChartType {
-  CASE_GROWTH_RATE = 'Infection rate',
```
Comment (goldblatt) [inline src/enums/zones.ts:5]: these shouldn't matter

File: src/screens/About/About.tsx [hunk 6]
```diff
@@ -21,6 +22,11 @@ const sidebar = (
 const About = ({ children }: { children: React.ReactNode }) => {
   return (
     <Wrapper>
+      <AppMetaTags
+        canonicalUrl="/about"
+        pageTitle="About Cvoid Act Now"
```
Comment (goldblatt) [inline src/screens/About/About.tsx:27]: Covid**

**Comment by goldblatt** [comment]
also i found the beta tag to be a lil off. kind of a nit
Screen Shot 2020-04-30 at 12 46 43 PM

**Comment by goldblatt** [comment]
and do we want to add the images of the charts to this PR since it already is updating the meta tags?

**Pull request merged**